### PR TITLE
Fixes #142 Grammatical errors and typos found in README.md

### LINKED ISSUE #142: Fix Grammatical mistakes and typos found in README.md
#### Problem description
There are some grammatical mistakes and typos in the README.md file.

#### Steps to reproduce the problem
Read the README.md file

### PR DESCRIPTION
**Pull request by rohap97**
Closes #142 

#### What has been done to verify that this works as intended?
This is just change in README.md so their is no working associated with it.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Since README.md file is where the users gather information about the working of the app, it is important that it is readable to the users. Thus, by removing grammatical mistakes it makes the README.md more user-friendly by making it more readable and understandable.

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by lakshyagupta21** [comment]
@rohap97 The readme changes looks good to me, please remove the extra changes that you had done in the PR and also resolve the conflicts if any.

**Comment by shobhitagarwal1612** [comment]
On a different note, @lakshyagupta21 this reminds me that we need to shorten the README and make it more readable. If possible, move some of the information to wiki pages.

**Comment by lakshyagupta21** [comment]
> On a different note, @lakshyagupta21 this reminds me that we need to shorten the README and make it more readable. If possible, move some of the information to wiki pages.

By Shorten the README, you mean to say move the usage instructions somewhere else? Are you suggesting to modify the ReadMe to make it similar to ODK-Collect Readme?

**Comment by rohap97** [comment]
@lakshyagupta21 Thankyou Sir, for reviewing my PR. I am new to Git and I don't know how to resolve merge conflicts. I have watched few youtube videos for the same, but it still is confusing to me. Any help on this issue will be really appreciated. Please help!

**Comment by iadeelzafar** [comment]
@rohap97 Follow this and if you've any issues feel free to ping me.
* git checkout master
* git remote add upstream https://github.com/opendatakit/skunkworks-crow.git - Run this command only if you have not added this remote before.
* git pull upstream master
* git checkout fix-readme
* git rebase master
* Now open Android Studio and you will see some conflicting lines, so resolve the conflicts.
* Commit and then force push the changes using git push -f origin fix-readme

**Comment by rohap97** [comment]
@iadeelzafar Thankyou, I followed your guide but after commit, when using "git push -f origin fix-readme". It says "Everything up-to-date". Any help regarding this? Enclosing the screenshot for your reference.

Merge Conflict

**Comment by shobhitagarwal1612** [comment]
You have to do `git rebase --continue` as well

**Comment by rohap97** [comment]
@shobhitagarwal1612 Sir, so my next line according to the above screenshot should be `git rebase --continue`?

**Comment by rohap97** [comment]
@shobhitagarwal1612 , @iadeelzafar Should I do `git reset --soft HEAD^` and then `git rebase --continue`. I found this link https://stackoverflow.com/questions/6457044/forgot-git-rebase-continue-and-did-git-commit-how-to-fix where they talk about this issue? Should I go ahead with the steps mentioned in the link?

**Comment by rohap97** [comment]
@shobhitagarwal1612 Sir, Thankyou very much for your support! Now this branch has no conflicts.

**Comment by rohap97** [comment]
@iadeelzafar Thank you very much, Because of you I learned about merging and solving branch conflicts! You have been a big help really!

**Comment by iadeelzafar** [comment]
@rohap97 I was outside, apologies for not being able to reply quickly. I think this worked for you. Is it?

**Comment by rohap97** [comment]
@iadeelzafar Yes, it worked out for me. Just a small query - By changing the branch name I can solve other merge conflicts using your Steps right? I have another PR #172  where there is the same issue of merge conflict.

**Comment by iadeelzafar** [comment]
@rohap97 Yep. Don't forget to read git's messages. Reading them properly will save you a lot of time and solve most of your problems.

**Comment by rohap97** [comment]
@iadeelzafar Should I not use `git remote add upstream https://github.com/opendatakit/skunkworks-crow.git` for solving the merging conflicts in other PR, or I can use it? Asking this question since you wrote "Run this command only if you have not added this remote before."

**Comment by iadeelzafar** [comment]
@rohap97 Once you've added it, there's no need to add it again.

**Comment by rohap97** [comment]
@iadeelzafar Thanks again!

File: skunkworks_crow/src/main/res/layout/toolbar.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     android:background="?attr/colorPrimary"
     android:minHeight="?attr/actionBarSize"
     app:contentInsetEnd="0dp"
-    app:contentInsetStart="0dp"
+    app:contentInsetStart="16dp"
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/res/layout/toolbar.xml]: Please undo this change as it is not related

File: skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java [hunk 2]
```diff
@@ -352,6 +352,9 @@ private void sendBlankForm(Cursor cursor) {
                 sbResult.append(getContext().getString(R.string.id, formId) + " " +
                         getContext().getString(R.string.success, getContext().getString(R.string.blank_form_count,
                                 getContext().getString(R.string.sent))));
+            } else {
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java]: Please undo this change as it is not unrelated

File: skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java [hunk 3]
```diff
@@ -52,7 +53,7 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onPageStarted(WebView view, String url, Bitmap favicon) {
                 super.onPageStarted(view, url, favicon);
-                getSupportActionBar().setTitle(url);
+                getSupportActionBar().setTitle(OPEN_SOURCE);
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java]: Undo this change please

File: skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java [hunk 4]
```diff
@@ -30,6 +30,7 @@
 public class WebViewActivity extends AppCompatActivity {
 
     public static final String OPEN_URL = "url";
+    public static final String OPEN_SOURCE = "Open Source Licenses";
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java]: Undo this change please

File: skunkworks_crow/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -123,4 +123,5 @@
     <string name="num_reviewed">Reviewed: %s</string>
     <string name="num_unreviewed">Unreviewed: %s</string>
     <string name="no_forms">No Forms Found</string>
+    <string name="form_available">already available at the receiver</string>
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/res/values/strings.xml]: Undo this change please

Comment (rohap97) [reply skunkworks_crow/src/main/res/values/strings.xml]: @shobhitagarwal1612 Sir, Undid it.

File: skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java (hunk 4 above)
Comment (rohap97) [reply skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java]: @shobhitagarwal1612 Sir, Undid it.

File: skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java (hunk 3 above)
Comment (rohap97) [reply skunkworks_crow/src/main/java/org/odk/share/activities/WebViewActivity.java]: @shobhitagarwal1612 Sir, Undid it.

File: skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java (hunk 2 above)
Comment (rohap97) [reply skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java]: @shobhitagarwal1612 Sir, Undid it.

File: skunkworks_crow/src/main/res/layout/toolbar.xml (hunk 1 above)
Comment (rohap97) [reply skunkworks_crow/src/main/res/layout/toolbar.xml]: @shobhitagarwal1612 Sir, Undid it.

File: README.md [hunk 6]
```diff
@@ -123,11 +123,11 @@ And for other device it will be enabled automatically.
...
 
-Tap on the “Receive” button shown at the bottom of the home screen to see a list of all wifi networks available in the nearby area which are started by only skunkworks-crow app.
+Tap on the 'Receive' button shown at the bottom of the home screen to see a list of all wifi networks available in the nearby area which are started by only skunkworks-crow app.
 
 A receiver will need to connect to the network the sender has created. There are two ways to connect to the network:
-A QR code generated by the sender can be scanned by a receiver, it will get all the information required to connect to the sender (wifi name, port,  and password if the connection is protected).
-Tap on the wifi network available to connect and after successful connection enter port number to start receiving the forms.
+A QR code generated by the sender that can be scanned by a receiver, it will get all the information required to connect to the sender (wifi name, port,  and password if the connection is protected).
+Or else, Tap on the wifi network available to connect and after successful connection enter port number to start receiving the forms.
```
Comment (shobhitagarwal1612) [inline README.md]: Tap should be tap

Comment (rohap97) [reply README.md]: @shobhitagarwal1612 Sir, Done

**Comment by shobhitagarwal1612** [comment]
Congratulations :tada: :tada: :tada:

**Comment by rohap97** [comment]
@shobhitagarwal1612 Thank you Sir. My Pull Request got merged because of @shobhitagarwal1612 and @iadeelzafar . Thank you very much both of you!
